scripts/utils.py

```diff
@@ -338,6 +338,42 @@ def on_key(element):
             writer.writerow(f)
 
 
+def get_indexd_batch(guids):
+    """
+    Get indexd records by batch
+    """
+    results = {}
+    indexd_client = IndexClient(
+        INDEXD["host"],
+        INDEXD["version"],
+        (INDEXD["auth"]["username"], INDEXD["auth"]["password"]),
+    )
+    resp = indexd_client.bulk_request(guids)
+
+    for doc in resp:
+        results[doc.did] = doc.urls
+
+    return results
+
+
+def get_indexd_record(guid):
+    """
+    Get indexd records by batch
+    """
+    indexd_client = IndexClient(
+        INDEXD["host"],
+        INDEXD["version"],
+        (INDEXD["auth"]["username"], INDEXD["auth"]["password"]),
+    )
+    resp = indexd_client.get(guid)
+    try:
```

Comment:
Would like some feedback on what is to be returned when there is no record, right now this seems to work fine

Reply:
I would say lets show in the logs what guid was not indexed. For now lets create a list of files that were not indexed and output in the end of the jobs.